net-mgmt/zabbix-proxy/src/opnsense/mvc/app/models/OPNsense/Zabbixproxy/General.xml

```diff
@@ -106,6 +106,27 @@
             <asList>Y</asList>
             <Required>N</Required>
         </statsip>
+        <vmwarecachesize type="TextField">
+            <default>8M</default>
```

Comment:
this needs a validation ;)

Reply:
yes but it's never applied in the model. I changed it to accept values. 8M should already be applied when nothing is given (like it is now).